tests/integration/test_sns.py

```diff
@@ -73,7 +73,7 @@ def check_message():
             msg_received = msgs["Messages"][0]
```

Comment:
Great changes @pinzon ! 🎉 Can we change
```
class SNSTest(unittest.TestCase):
```
to
```
class SNSTest:
```
(Then we can also remove the `unittest` import entirely). We'll also need to replace the `setUpClass` and `tearDownClass` with [pytest fixtures](https://docs.pytest.org/en/6.2.x/fixture.html) (e.g., using a `@pytest.fixture(scope='class')` decorator).